### PR TITLE
 Create an element with a child or children 

### PR DESCRIPTION
**Pull request by DavidBruant**
(Replacing #14)

- [ ] Must enable creating an element with a child with a nested syntax (`Bouture.table.tbody.tr`)
- [ ] Must enable creating an element with a single child passed as an argument in an array

### CODE REVIEW

**Review by pointbar — APPROVED** [review]

**Comment by DavidBruant** [comment]
Changed the PR name to steal #14's name. I had closed this issue and reported the change in the various github-projects but the name of this PR had remained the name of the commit

File: test/bouture.test.js [hunk 1]
```diff
@@ -222,10 +222,29 @@ describe(`Bouture.table.tbody`, () => {
...
+    const h1 = Bouture.h1()
+    const divDivA = Bouture.div().div('').a(`{href: 'http://bouture.com'}`)
+
+    expect(divA.getElement().tagName.toLowerCase()).to.equal('div')
+    expect(h1.getElement().tagName.toLowerCase()).to.equal('h1')
+    expect(divDivA.getElement().tagName.toLowerCase()).to.equal('div')
+
+    expect(divA.getElement().children.length).to.equal(1)
+    expect(h1.getElement().children.length).to.equal(1)
+    expect(divDivA.getElement().children.length).to.equal(2)
```
Comment (DavidBruant) [inline test/bouture.test.js]: Warning here, it has only one child not 2 (but this child also has a child)

File: test/bouture.test.js [hunk 2]
```diff
@@ -222,10 +222,29 @@ describe(`Bouture.table.tbody`, () => {
...
+    const divA = Bouture.div().p()
+    const h1 = Bouture.h1()
+    const divDivA = Bouture.div().div('').a(`{href: 'http://bouture.com'}`)
+
+    expect(divA.getElement().tagName.toLowerCase()).to.equal('div')
+    expect(h1.getElement().tagName.toLowerCase()).to.equal('h1')
+    expect(divDivA.getElement().tagName.toLowerCase()).to.equal('div')
+
+    expect(divA.getElement().children.length).to.equal(1)
+    expect(h1.getElement().children.length).to.equal(1)
```
Comment (DavidBruant) [inline test/bouture.test.js]: It looks like h1 should have 0 children?

**Comment by DavidBruant** [comment]
Let's close this PR and experiment elsewhere. It can be reopened later

**Pull request closed without merging**